src/core/dbus-execute.c

```diff
@@ -94,6 +95,10 @@ static int property_get_environment_files(
         return sd_bus_message_close_container(reply);
 }
 
+static bool oom_score_adjust_is_valid(int oa) {
+        return oa >= OOM_SCORE_ADJ_MIN && oa <= OOM_SCORE_ADJ_MAX;
+}
```

Comment:
hmm, this looks the wrong way round. i think parse_oom_score_adjust(),  should just use this helper too